src/main/java/edu/kis/vh/nursery/DefaultCountingOutRhymer.java

```diff
@@ -0,0 +1,40 @@
+package edu.kis.vh.nursery;
+
+public class DefaultCountingOutRhymer {
+
+	private static final int MAX_CAPACITY = 12;
+	private static final int STACK_EMPTY_VALUE = -1;
+	private final int[] NUMBERS = new int[MAX_CAPACITY];
+
+	public int getTotal() {
+		return total;
+	}
+
+	private int total = STACK_EMPTY_VALUE;
```

Comment:
Extracting this constant is done incorrectly. Should show during next classes ;)